core/src/main/java/com/orange/ouds/core/component/OudsButton.kt

```diff
@@ -85,6 +89,8 @@ import kotlinx.parcelize.Parcelize
  * Note that in the case it is placed in an [OudsColoredBox], its monochrome variant is automatically displayed.
  * The tokens associated with these specific colors can be customized by overriding [OudsButtonMonoTokens].
  *
+ * Rounded corners can be enabled or disabled using the [OudsTheme.Settings.roundedCorners] property of an [OudsTheme.Settings] when calling the [com.orange.ouds.core.theme.OudsTheme] method.
```

Comment:
Don't forget to update this comment with `buttonRoundedCorner` property. Same thing for other composables.